app/models/veteran.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+require 'evss/auth_headers'
+
+# Veteran model
+class Veteran < Common::Base
+
+  attribute :veteran_name
+  attribute :user_poa_info_available
+  attribute :can_be_validated_by_group_one
+  attribute :award_suspendbadaddress_indicator
+  attribute :award_suspend_indicator
+  attribute :social_security_number
+  attribute :fiduciary_ind
+  attribute :has_payment
+  attribute :incompetent_ind
+  attribute :indentity_ind
+  attribute :index_ind
+
+  attr_accessor :power_of_attorney
+
+  def initialize(user)
+    @user = user
+    client = EVSS::CommonService.new(auth_headers)
+    build_from_json(client.get_current_info)
+  end
+
+  def build_from_json(json_data)
+    json_data = json_data.deep_transform_keys { |key| key.to_s.underscore }
+    json_data['info'].each { |key, value| send("#{key}=", value) if respond_to?("#{key}=") }
```

Comment:
This model reads well, but maybe consider a comment on this line. I'm generally a fan of as little comments as necessary so if I'm the only reviewer that is a bit confused by this then there's probably no need to add a comment. I'm guessing that you are copying all the entries in info into attributes? If that's the case it might be a good candidate for a method like `set_as_attribute` to convey intention .. I don't feel super strongly about this, just throwing out some ideas

Reply:
If this was an active record model, I could just use assign attributes, but that appears to be missing from active model. I can break it out though.

Reply:
I broke this out and added a comment

Reply:
Done